### PR TITLE
ascan alpha help tweaks

### PR DESCRIPTION
**Pull request by kingthorin**
ascanalpha.html > Sorted help content alphabetically, adjusted names to match scan rules, and added missing entries.
ZapAddOn.xml > Updated changes section.

### CODE REVIEW

File: src/org/zaproxy/zap/extension/ascanrulesAlpha/resources/help/contents/ascanalpha.html [hunk 1]
```diff
@@ -52,22 +52,36 @@ <H2>Httpoxy</H2>
...
 This active scanner attempts to access content that was originally accessed via HTTPS (SSL/TLS) via HTTP.
 
 <H2>LDAP Injection</H2>
 LDAP Injection may be possible. It may be possible for an attacker to bypass authentication controls, and to view and modify arbitrary data in the LDAP directory.
 
-<H2>SQL Injection - MsSQL</H2>
-This active scanner attempts to inject MsSQL specific sleep commands into parameter values and analyzes the server's response time to see if the sleep is effectively executed on the server (indicating a successful SQL injection attack).
+<H2>Proxy Disclosure</H2>
+Attempts to detect and fingerprint proxy server(s). This information helps a potential attacker to determine.<br>
+- A list of targets for an attack against the application.<br>
```
Comment (psiinon) [inline src/org/zaproxy/zap/extension/ascanrulesAlpha/resources/help/contents/ascanalpha.html]: Any reason not to use <ul><li>... for these?

Comment (kingthorin) [reply src/org/zaproxy/zap/extension/ascanrulesAlpha/resources/help/contents/ascanalpha.html]: Nope. Will change it.

File: src/org/zaproxy/zap/extension/ascanrulesAlpha/resources/help/contents/ascanalpha.html [hunk 2]
```diff
@@ -9,28 +9,30 @@
 <H1>Active Scan Rules - alpha</H1>
 The following alpha quality active scan rules are included in this add-on:
 
+<H2>An example active scan rule which loads data from a file</H2>
+This implements an example active scan rule that loads strings from a file that the user can edit.<br>
+For more details see: 
+<a href="http://zaproxy.blogspot.co.uk/2014/04/hacking-zap-4-active-scan-rules.html">Hacking Zap Part 4: Active Scan Rules.</a>
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/ascanrulesAlpha/resources/help/contents/ascanalpha.html]: I'd suggest ZAP all upper case (same comment for the other link).

**Comment by thc202** [comment]
Should `.htaccess Information Leak` be moved to top?

**Comment by kingthorin** [comment]
Yup I'm good with that.

**Comment by kingthorin** [comment]
Done

**Comment by thc202** [comment]
Thanks!

**Comment by thc202** [comment]
New release?

**Comment by kingthorin** [comment]
I don’t have anything else in mind right now, so sure from my perspective.

**Comment by psiinon** [comment]
:+1:

**Pull request merged**